### PR TITLE
Front end and back end setup

### PR DESCRIPTION
**Pull request by tamjankowska**

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (nmunro) [inline frontend/.gitignore]: Is this needed, since there's a .gitignore at the top level?

Comment (tamjankowska) [reply frontend/.gitignore]: I was following the structure of the mern app... we didn't remove it from that so I didn't really think about it. I can see why it wouldn't be necessary though, shall I delete it?

**Review by nmunro — APPROVED** [review]
I approve my code! 👍

**Review by maddyharrop — APPROVED** [review]

**Review by Joe-Bentley — APPROVED** [review]
If Neil approves, I approve

**Pull request merged**